src/components/CreatePartyPage/Form/Form_footer/Checkout/index.js

```diff
@@ -0,0 +1,162 @@
+                Fortsätt som inloggad användare
+                </label>
+            </div>
+            <div className="input-group">
+              <input
+                className="radio-input"
+                id="radio4"
+                name="radio"
+                type="radio"
+                onClick={this.registerToggle}
+              />
+              <label className="radio-label" htmlFor="radio4">
+                Fortsätt utan inloggning
+                </label>
+            </div>
+          </div>
+        </div>
+
+
+        {this.state.loginOption ? (
+          <div>
+            <div className="login-container">
+              <div className="login-content">
+                <h2 className="form-headline">Skapa konto</h2>
+                <FormGroup >
+                  <Label htmlFor="email-input">E-postadress</Label>
+                  <Input type="email" name="email" id="email-input" placeholder="E-postadress" className="registration-form" />
+                </FormGroup>
+                <FormGroup>
+                  <Label htmlFor="password-input">Nytt lösenord</Label>
+                  <Input type="password" name="password" id="password-input" placeholder="Nytt lösenord" pattern="^((?=.*[a-z])(?=.*[A-Z])(?=.*\d)|(?=.*[a-z])(?=.*[A-Z])(?=.*[^A-Za-z0-9])|(?=.*[a-z])(?=.*\d)(?=.*[^A-Za-z0-9])|(?=.*[A-Z])(?=.*\d)(?=.*[^A-Za-z0-9]))([A-Za-z\d@#$%^&amp;*\-_+=[\]{}|\\:',?/`~&quot;();!]|\.(?!@)){8,16}$" title="Välj ett nytt lösenord" required="" className="registration-form" />
+                </FormGroup>
+                <div className="error item-level" aria-hidden="true"><p className="registration-text">Lösenordet måste bestå av 8-16 tecken, och innehålla minst 3 av följande alternativ: Stor eller liten bokstav (A-z), siffra eller specialtecken.</p></div>
+                <FormGroup>
+                  <Label htmlFor="password-input"></Label>
+                  <Input type="password" name="password" id="password-confirmation-input" placeholder="Bekräfta ditt nya lösenord" pattern="^((?=.*[a-z])(?=.*[A-Z])(?=.*\d)|(?=.*[a-z])(?=.*[A-Z])(?=.*[^A-Za-z0-9])|(?=.*[a-z])(?=.*\d)(?=.*[^A-Za-z0-9])|(?=.*[A-Z])(?=.*\d)(?=.*[^A-Za-z0-9]))([A-Za-z\d@#$%^&amp;*\-_+=[\]{}|\\:',?/`~&quot;();!]|\.(?!@)){8,16}$" title="Upprepa ditt nya lösenord" required="" className="registration-form" />
+                </FormGroup>
+                <div className="error item-level login-item" aria-hidden="true"><p className="registration-text mb-2">Har du redan ett konto? Vänligen logga in.</p></div>
+                <Button color="primary" type="button" onClick={this.loginToggle} >Avbryt</Button>
+                <Button color="primary" type="button" className="ml-lg-2" >Fortsätt</Button>
+                <Button color="primary" type="button" className="ml-lg-2" onClick={this.userLoginToggle}>Logga in</Button>
+              </div >
+            </div>
+          </div>) : null}
+
+
```

Comment:
Finns många saker här som kan mappas ut eller att du skapar separata komponenter så det inte blir 200 rader kod då detta inte kan återanvändas samt kör inte conditional rendering direkt i din JSX gör det utanför som det är gjort i imageHandler, tex:

randomFunction = () => this.state.loginOption ? (
     
           jsx här 

)

sen rendera ut genom att skriva:

{this.randomFunction()}

Reply:
jag ska. Vil start ny branch där jag ska mappa ut och ha redux för gästanvändare

Reply:
Arrajt gör en ny branch på denna branch då så blire perfa :)